src/tests/integration/test-data/foundry/test/FreshInt.t.sol

```diff
@@ -30,4 +33,32 @@ contract FreshIntTest is Test, KEVMCheats {
         int128 val = int128(uint128(kevm.freshUInt(16)));
         assertGt(val, max);
     }
+
+    function manip_symbolic_bytes(bytes memory b) public {
+        uint middle = b.length / 2;
+        b[middle] = hex'aa';
+    }
+
+    function test_symbolic_bytes_function() public {
+        bytes memory fresh_bytes = kevm.freshBytes(5);
+        manip_symbolic_bytes(fresh_bytes);
+        assertEq(hex'aa', fresh_bytes[2]);
+    }
+
+    function test_symbolic_bytes_1() public {
+        bytes memory fresh_bytes_1 = kevm.freshBytes(5);
+        assertEq(fresh_bytes_1.length, 5);
+        local_byte = fresh_bytes_1[0];
```

Comment:
How is this variable used?

Reply:
I was just trying to test out manipulations of the generated symbolic byte array, including copying a single byte of the symbolic array to storage (`local_byte`) and copying the entire array to storage (`local_bytes`)

Reply:
I see, ok. Asserting the length equality is not needed since there is a separate, more general test for that below (` test_symbolic_bytes_length`). Could we assert somehow that the copying worked?

Reply:
What I've done for now is assert that the variables `fresh_bytes_2` and `local_bytes` are equal.